src/Client/AbstractClient.php

```diff
@@ -16,7 +16,7 @@ abstract class AbstractClient
     /**
      * The Collmex API endpoint URL template
      */
-    const EXCHANGE_URL = 'https://www.collmex.de/cgi-bin/cgi.exe?%s,0,data_exchange';
+    public const EXCHANGE_URL = 'https://www.collmex.de/cgi-bin/cgi.exe?%s,0,data_exchange';
```

Comment:
@mjaschen Should we make this one private?

Reply:
I've no opinion on that question 😐